cypress/integration/group1/hostedWorkflows.ts

```diff
@@ -146,5 +148,53 @@ describe('Dockstore hosted workflows', () => {
       goToTab('Files');
       cy.contains('/Dockstore.wdl').should('be.visible');
     });
+    it('Create a new hosted workflow', () => {
+      cy.get('#registerWorkflowButton').should('be.visible').should('be.enabled').click();
+      cy.get('#3-register-workflow-option').should('be.visible').click();
+      cy.contains('button', 'Next').click();
+      cy.get('#hostedWorkflowRepository').type(NEW_WORKFLOW_NAME);
+      cy.contains('button', 'Register Workflow').click();
+    });
+    it('Add files to hosted workflow', () => {
+      // navigate to workflow
+      cy.get('.mat-expanded');
+      cy.contains('dockstore.org/user_A').click();
+      cy.contains('a', NEW_WORKFLOW_NAME).click();
+
+      // Check content of the info tab
+      cy.contains('Mode: HOSTED');
+
+      // Add a new version with 3 descriptors
+      goToTab('Files');
+      cy.get('#editFilesButton').click();
+
+      // there should be no descriptors files
+      cy.get('app-code-editor').should('have.length', 0);
+
+      // add first file. This will be the primary descriptor, so we wont need to give it a custom name
+      cy.contains('Add File').click();
+
+      // add second file
+      cy.contains('Add File').click();
+      cy.wait(100);
+      cy.get('.editor-file-name').last().type('{backspace}{backspace}{backspace}{backspace}{backspace}/AAA.cwl{enter}');
```

Comment:
Assuming these backspaces are due to the default names like `/.cwl` this becomes `/AAA.cwl`
Is it possible to make this more readable by looping through the length of a string or even better the actual constant in that we use to populate new files?

Reply:
Cheated a bit, Cypress provides `{selectall}` to select all content in the current input bar and paired that with a single `{backspace}`. Also moved the file creations into a loop, no reason to have those lines duplicated.